apps/api/src/modules/report/report.repository.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@nestjs/common'
+import { PrismaService } from 'src/database/prisma.service'
+
+import { ReportPostData } from './report.type'
+
+@Injectable()
+export class ReportRepository {
+  constructor(private prisma: PrismaService) {}
+
+  async createReport(payload: ReportPostData) {
+    const { jobID, userID, reason } = payload
+    await this.prisma.report.create({
+      data: {
+        jobId: jobID,
+        reporterId: userID,
+        reason: reason,
+      },
+    })
```

Comment:
This should return created report to get its id from service, or just return reportId